scripts/helpers/check-mailbox.js

```diff
@@ -15,12 +15,13 @@ var augur = new Augur();
 augur.connect(connectionEndpoints, function (err) {
   if (err) return console.error(err);
   augur.api.Market.getMarketCreatorMailbox({ tx: { to: marketId } }, function (err, marketMailboxAddress) {
-    if (err) return console.log(err);
+    if (err) return console.error(err);
     console.log(chalk.green.dim("market mailbox address:"), chalk.green(marketMailboxAddress));
     augur.api.Cash.getBalance({ _address: marketMailboxAddress }, function (err, cashBal) {
-      if (err) return console.log(err);
+      if (err) return console.error(err);
       var cashBalance = speedomatic.bignum(cashBal);
-      augur.rpc.eth.getBalance([marketMailboxAddress, "latest"], function (attoEthBalance) {
+      augur.rpc.eth.getBalance([marketMailboxAddress, "latest"], function (err, attoEthBalance) {
+        if (err) return console.error(err);
```

Comment:
Should add check like: 
```
if (attoEthBalance == null) return callback(new Error("rpc.eth.getBalance failed"));
```

Reply:
Hm...the only thing that was updated in this PR was the callback signature (and changing console.log to console.error).  I think if the result that comes back is `null`, ethrpc will pass back an error object for `err`.